src/sql/connection.py

```diff
@@ -337,6 +337,7 @@ def _close(cls, descriptor):
     def close(self):
         self.__class__._close(self)
 
+    @classmethod
```

Comment:
if this is a class method, the first argument should be cls, by convention

Reply:
Fix